IFC4x3/Constants/g/GEOTECHTYPICALSECTION_1V8YeK3DXAxwY3SVvMcwhP.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="GEOTECHTYPICALSECTION_1V8YeK3DXAxwY3SVvMcwhP" Name="GEOTECHTYPICALSECTION" UniqueId="5f222a14-0cd8-4aef-a883-71fe569baad9">
+	<Documentation>Interval along the alignment/built structure with similar ground conditions, as part of the GeotechSynthesis model that represents the connection between the ground model and the built structure. Includes key-properties like expected distribution of ground types (reference to GeotechUnits) and baseline-definition of expected ground conditions and potential hazards, and may also include key-information on design like excavation measures, distribution of support types etc. These properties can be provided using pre- or user-defined property sets, using _IfcRelDefinesByProperties_ or external datasets using _IfcRelAssociatesDataset_.</Documentation>
```

Comment:
GeotechSynthesis and GeotechUnits are apparently concepts of some sort. There are no references to them. What do they mean?

Reply:
I think that, to be sure, we will need to go back to the domain team. There is an IfcGeoScienceModel/GEOTECHSYNTHESISMODEL and an IfcGeoScienceFeature/GEOTECHUNIT, but are they really meaning this in a strict sense?